dependency-check-maven-config.xml

```diff
@@ -164,4 +164,22 @@
         </notes>
         <cve>CVE-2016-8739</cve>
     </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs are all related to asciidoc and jruby vulnerabilities which do not
```

Comment:
✏️  You could also say something like, these are only used during the building of the documentation, just to be more specific.